docs/content/topics/operators/deploy.md

```diff
@@ -11,4 +11,429 @@ aliases:
+production-grade Brigade deploy.  This includes:
+
+  - [Managing external access to Brigade via a DNS hostname and ingress resource](#managing-inbound-traffic-via-ingress)
+  - [Utilizing a third-party certificate provider for public-facing TLS](#certificates-and-tls)
+  - [Choosing a third-party authentication provider for managing user auth](#third-party-authentication)
+
+(Alternatively, if you're interested in using Brigade in a local development
+environment or trial capacity, take a look at the [QuickStart].)
+
+[QuickStart]: /intro/quickstart.md
+
+## Prerequisites
+
+* A [Kubernetes cluster], version 1.18.8 or higher.
+  Your cluster must be accessible to the outside world, e.g. to third-party
+  authentication providers and source(s) of event triggers.
+  As long as the cluster is able to provision a public ip address, as any
+  cloud provider's Kubernetes offering will support, you should be good to go.
+* [Helm] CLI v3.3.1+ installed.
+* [kubectl] CLI installed.
+* Free disk space on the cluster nodes.
+  The installation requires sufficient free disk space and will fail if a cluster node disk is nearly full.
+
+[Kubernetes cluster]: https://kubernetes.io/docs/setup/
+[Helm]: https://helm.sh/docs/intro/install/
+[kubectl]: https://kubernetes.io/docs/tasks/tools/#kubectl
+
+## Prepping the Brigade 2 installation
+
+For now, we're using the [GitHub Container Registry](https://ghcr.io) (which is
+an [OCI registry](https://helm.sh/docs/topics/registries/)) to host our Helm
+chart. Helm 3 has _experimental_ support for OCI registries. In the event that
+the Helm 3 dependency proves troublesome for Brigade users, or in the event that
+this experimental feature goes away, or isn't working like we'd hope, we will
+revisit this choice before going GA.
+
+First, let's set the necessary environment variable to enable this experimental
+support and pull the Brigade chart to a local directory:
+
+```console
+$ export HELM_EXPERIMENTAL_OCI=1
+$ helm chart pull ghcr.io/brigadecore/brigade:v2.0.0-beta.1
+$ helm chart export ghcr.io/brigadecore/brigade:v2.0.0-beta.1 -d ~/charts
+```
+
+We're now ready to view and edit configuration options:
+
+```console
+$ helm inspect values ~/charts/brigade > ~/brigade2-values.yaml
+```
+
+In the next steps, we'll go through the configuration options needed to
+acheive the objectives stated at the start of this document.
+
+## Managing inbound traffic via Ingress
+
+Before configuring Brigade to create an [Ingress resource] for its API server,
+we need to deploy an [Ingress Controller] as well as create a DNS hostname
+record which will map to the Ingress controller's external IP address.
+
+[Ingress resource]: https://kubernetes.io/docs/concepts/services-networking/ingress/
+[Ingress Controller]: https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/
+
+### Deploying the Ingress Controller
+
+There are myriad Ingress Controller options.  For this guide, we will use the
```

Comment:
```suggestion
There are many Ingress Controller options.  For this guide, we will use the
```

Love the word "myriad," but given the likely number of readers for whom English may be a second language, I think "many" will be more widely understood.